frontend.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": [
+      "github>empathyco/.github",
+      ":semanticCommitTypeAll(chore)",
+      "schedule:earlyMondays"
```

Comment:
Should we add `"schedule:earlyMondays",` as default and, then.. could it be changed by some repos?

Reply:
yes, that's the idea!